R/convertFromBIOM.R

```diff
@@ -230,18 +225,33 @@ convertFromBIOM <- function(
         assays = assays,
         colData = sample_data,
         rowData = feature_data)
+    
+    # Set ranks based on rowData columns if user has specified to do so
+    temp <- .set_ranks_based_on_rowdata(tse, ...)
```

Comment:
Now importBIOM supports also setting new ranks based on data. In theory, the file can include more ranks that are supported by mia